HazardScout/Recruiting-Project
# pm(feat): recruiting-project-nyan-cat

## Changes

- Designed according to **Project Requirements** section of README

## Testing Steps

- Notice table with list of guest and caption containing total tickets count and guest count by email address
- Click **Add Guest** button to open modal component
- Click **OK** without a valid email and/or with a very high amount of tickets such that the total tickets > 20 to test form validation
- Fill out form inside modal with valid values and click **OK** and notice guest is added to list
- Click edit icon to open same modal component
- Update the values and click **Cancel**
- Notice original values in table were not modified
- Edit again, set valid values and click **OK**
- Notice edited value is updated in table
- Click trash icon to remove a guest from the list
- Notice confirmation modal
- Ensure guest is only removed from list when **OK** is selected
- Delete all guest to view empty list message under table
- Notice that a new guest cannot be added if total ticket count is 20

[HazardScout/Recruiting-Project](https://github.com/HazardScout/Recruiting-Project)
